src/layers/TrackerLayer.js

```diff
@@ -347,24 +350,32 @@ class TrackerLayer extends Layer {
 
       ctx.beginPath();
       ctx.arc(origin, origin, radius, 0, 2 * Math.PI, false);
-      ctx.fillStyle = color || getBgColor(type);
-      ctx.fill();
+      if (!this.useDelayStyle) {
+        ctx.fillStyle = color || getBgColor(type);
+        ctx.fill();
+      } else if (this.useDelayStyle && delay === null) {
```

Comment:
I don't think you need this 'else' jsut adapt getDelayColor to return gray when delay is null